app/Traits/ClearsGlobalSearchCache.php

```
@@ -10,77 +10,119 @@ trait ClearsGlobalSearchCache
             }
-        }
 
-        return false;
+            return false;
+        } catch (\Throwable $e) {
+            // If checking changes fails, assume changes exist to be safe
+            ray('Failed to check searchable changes: '.$e->getMessage());
+
+            return true;
+        }
     }
```

PHPMD (2.15.0)

The method hasSearchableChanges() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)